src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -630,50 +630,59 @@ public BundleContext getBundleContext()
      */
     public AbstractEndpoint getEndpoint(String id)
     {
-        return getEndpoint(id, /* create */ false);
+        if (id == null)
```

Comment:
nit: How about `Objects.requireNonNull(id)`?

Reply:
and/or tag `id` with `NotNull`

Reply:
Done. Thank you very much for review!